src/Distributions.jl

```diff
@@ -239,8 +241,10 @@ export
 
     # reexport from StatsBase
     sample, sample!,        # sample from a source array
-    wsample, wsample!      # weighted sampling from a source array
+    wsample, wsample!,      # weighted sampling from a source array
 
+    # reexport from LinearAlgebra
+    rank
```

Comment:
do we need to re-export stdlib functions?

Reply:
Do you consider this different from re-exporting `mean` and so on from `Statistics`?